transcom/mymove
# B-17490 Add Deliveries Table to Database

## [DB - Add Partial Delivery Table](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A842079&RoomContext=TeamRoom%3A808731&concept=TeamRoom)

## Summary

Adding a table to the database called `deliveries` that will be responsible for handling full and partial deliveries for service items and will be used to calculate weight delivered to be compared to the `shipments` table, specifically the `prime_actual_weight` column. 

### How to test

1. Pull the branch
2. Run `make db_dev_migrate`
3. Confirm that the `deliveries` table appears in the database, as well as added comments
